solutions/level1/main.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'cart'
+require_relative 'library'
+require_relative 'book_serializer'
+
+FILE = '../../requirements/level1/data.json'
+
+library = Library.new
+books_hash = JsonReaderWriter.read_file(FILE)
```

Comment:
I think it might be a more OOP solution to read and create the books when initializing the Library. It's the responsibility of the library